introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb

```diff
@@ -44,8 +44,9 @@
     "3. [Fine-tune the pre-trained model on a custom dataset](#3.-Fine-tune-the-pre-trained-model-on-a-custom-dataset)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        "mxnet-od-ssd-512-vgg16-atrous-coco",  # "pytorch-od1-fasterrcnn-resnet50-fpn"
</pre><p>Delete the commented-out <code>pytorch-od1-fasterrcnn-resnet50-fpn</code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
This is commented out in purpose, it is an alternative model-id that supports re-training, that was missing from the original Notebook.